### PR TITLE
JENA-1495 Return Model from PrefixMapping methods.

### PR DESCRIPTION
**Pull request by jaco0646**

### CODE REVIEW

File: jena-core/src/main/java/org/apache/jena/rdf/model/Model.java [hunk 1]
```diff
@@ -1051,4 +1051,12 @@ Remove all the statements matching (s, p, o) from this model.
         Answer true iff .close() has been called on this Model.
     */
     public boolean isClosed();
+
+    // Override return type for methods inherited from PrefixMapping
```
Comment (ajs6f) [inline jena-core/src/main/java/org/apache/jena/rdf/model/Model.java]: We should use `@Override` for explicitness on these changes.

**Comment by ajs6f** [comment]
This seems like a useful change, but it is not back-compatible, so in theory we should hold it for a major version release.

**Comment by ajs6f** [comment]
This looks basically good to me: with an additional commit addressing https://github.com/apache/jena/pull/368#pullrequestreview-101006841, I'll be happy to merge this.

File: jena-core/src/main/java/org/apache/jena/rdf/model/Model.java (hunk 1 above)
Comment (jaco0646) [reply jena-core/src/main/java/org/apache/jena/rdf/model/Model.java]: Done.

**Comment by afs** [comment]
Closing this PR. I see these commits in the codebase under different SHA1 ids (b8b482ecaa and 036ebce8ed).

@jaco0646 - thanks!

**Pull request closed without merging**